docs/api-reference/base-layer.md

```diff
@@ -204,17 +204,49 @@ manually override it using this prop.
 
 ##### `updateTriggers` (Object, optional)
 
-This prop expects an object of which the keys matching the accessor names of a layer.
-If any values in this object are changed between props update, the attribute corresponding
-to the accessors named by the key will be invalidated.
+Accessors such as `getColor` and `getPosition` are called to retrieve colors
+ and positions when a layer is first added. From then on, to maximize performance, 
+ deck.gl does not recalculate colors or positions unless the `data` prop changes
+ by shallow comparison.
+
+Sometimes `data` remains the same, but the outcome of an accessor has changed.
+ In the following example, this is caused by changes in the external values 
+ `maleColor` and `femaleColor`:
+```
+  const layer = new ScatterplotLayer({
+    ... // Other props
+    getColor: d => d.male ? maleColor : femaleColor
+  });
+```
+
+In this case, you need to explicitly inform deck.gl to re-evaluate `getColor` 
+ for all data items. You do so by defining `updateTriggers`:
+```
+  const layer = new ScatterplotLayer({
+    ... // Other props
+    getColor: d => d.male ? maleColor : femaleColor,
+    updateTriggers: {
+        getColor: [maleColor, femaleColor]
+    }
+  });
+```
+
+`updateTriggers` expect an object whose keys are names of accessor props of this layer,
```

Comment:
lets remove mention of `object` for API clarity, even if it's still supported under the hood. Instead, `a value or array of values` should suffice.